packages/os/healthdog.service

```diff
@@ -0,0 +1,7 @@
+[Unit]
+Description=Send a Healthdog Ping
+
+[Service]
+Type=oneshot
+RemainAfterExit=false
```

Comment:
Why false?  We usually have this be true so that the service doesn't look unhealthy if it ran OK the last time.

Reply:
`true` definitely did not work with the timer. I don't pretend to be a systemd expert, but I think RemainAfterExit makes it seem like the service is 'active' and the timer doesn't start it again.

Reply:
Ah, ok, I don't have much experience with timers :)